packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js

```diff
@@ -51,12 +51,70 @@ moduleFor(
       this.addTemplate(
         'application',
         strip`
-      <ul>
-        {{#each model as |item|}}
-          <li>{{item}}</li>
-        {{/each}}
-      </ul>
-    `
+        <ul>
+          {{#each this.model as |item|}}
+            <li>{{item}}</li>
+          {{/each}}
+        </ul>
+        `
+      );
+
+      return this.visit('/').then(() => {
+        this.assertInnerHTML(strip`
+        <ul>
+          <li>red</li>
+          <li>yellow</li>
+          <li>blue</li>
+        </ul>
+      `);
+      });
+    }
+
+    ['@feature(!EMBER_ROUTING_MODEL_ARG) it cannot access the model provided by the route via @model']() {
+      this.add(
+        'route:application',
+        Route.extend({
+          model() {
+            return ['red', 'yellow', 'blue'];
+          },
+        })
+      );
+
+      this.addTemplate(
+        'application',
+        strip`
+        <ul>
+          {{#each @model as |item|}}
```

Comment:
Do we expect using a named arg that is not provided to error or anything? Seems like we don't, right?

Reply:
correct, we do not, it would just be undefined